.codecov.yaml

```diff
@@ -0,0 +1,12 @@
+# Summary: configuration for codecov/codecov-action used in ci.yml.
+# See https://github.com/codecov/codecov-action for info about the GitHub Action.
+# See https://app.codecov.io/gh/quantumlib/Cirq for the results dashboard.
```

Comment:
nit - consider deleting this line so the config file can be copied to other projects without introducing off-topic link for cirq.
```suggestion
```